Cargo.toml

```diff
@@ -18,3 +18,9 @@ panic = "abort"
 codegen-units = 1
 lto = true
 panic = "abort"
+
+[patch.crates-io]
+event-manager = { git = "https://github.com/alexandruag/event-manager.git", branch = "fix" }
```

Comment:
Can you please post the PR in event-manager as well? We already have a few forks that we need to keep track of.

Reply:
Will do after I finish answering the other comments here.